ReactTask/src/components/BtnsComponents/BtnSubmitFormReset.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { StylIconArrowBtn } from './../common/icons/';
+import styled from 'styled-components';
+import { StylBtnForm } from './../common/Btns/';
+
+const BtnForm = styled(StylBtnForm)`
+  max-width: 198px;
+`;
+
+const BtnSubmitFormReset = ({ text }) => {
+  return (
+    <BtnForm>
+      {text}
+      <StylIconArrowBtn />
+    </BtnForm>
+  );
+};
+
+export default BtnSubmitFormReset;
```

Comment:
Code duplicate with BtnSubmitForm.js

Reply:
The max - width style is added to the formReset button

Reply:
Buttons dont have to have width, any button has with according to content. Create one component for all buttons.

Reply:
Created 1 component `btnSubmitForm`